.github/workflows/reusable-release-policy-catalog.yml

```diff
@@ -57,7 +57,8 @@ jobs:
             PAYLOAD=$(echo "$PAYLOAD" | jq --arg policy_working_dir "${{ inputs.policy-working-dir }}" '.client_payload.policy_working_dir = $policy_working_dir')
           fi
 
-          echo "payload=$PAYLOAD" >> payload.json
+          echo "$PAYLOAD"
+          echo "$PAYLOAD" >> payload.json
```

Comment:
Ah! That's explain a lot of things... xD

Reply:
yes.. :face_exhaling:  I facepalmed myself hard IRL.